health-services/project-factory/src/server/utils/genericUtils.ts

```
@@ -664,54 +739,89 @@ async function createFacilityAndBoundaryFile(facilitySheetData: any, boundaryShe
+    logger.info("No dropdowns found");
+    receivedDropdowns= setDropdownFromSchema(request,schema,localizationMap);
+    logger.info("refetched drodowns",JSON.stringify(receivedDropdowns))
+  }
+  await handledropdownthings(facilitySheet, receivedDropdowns);
+  await handleHiddenColumns(facilitySheet, request.body?.hiddenColumns);
 
   // Add boundary sheet to the workbook
   const localizedBoundaryTab = getLocalizedName(getBoundaryTabName(), localizationMap);
   const boundarySheet = workbook.addWorksheet(localizedBoundaryTab);
-  addDataToSheet(boundarySheet, boundarySheetData, 'F3842D', 30, false, true);
+  boundarySheetData = modifyBoundaryIfSourceMicroplan(boundarySheetData, request);
```

Biome

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)